### ISSUE TITLE
Issue with Git

### ISSUE THREAD

**Issue by Baptiste-Garcin** [sickill/stderred#61]
I am testing stderred and for now I am globally satisfied.
There is still an issue with git...
Git push return is colored in red even if everything went ok as shown on screenshots...

![screenshot from 2017-09-13 14-25-44](https://user-images.githubusercontent.com/12398583/30377404-e53e165a-988f-11e7-8c99-d9ed0bc4b4aa.png)

Other point, when I type an unknown command, it does not return anything on red. Maybe it's an expected comportent but still...

![screenshot from 2017-09-13 14-28-54](https://user-images.githubusercontent.com/12398583/30377410-e998a422-988f-11e7-859d-8b5b39682fb3.png)

Thanks for your plugin anyway

**Comment by devnoname120** [comment]
Yeah, stderred colors stderr in red. Git outputs that stuff to stderr hence the issue. This can’t be fixed on our side, you can change all the coloring to e.g. yellow instead of red by defining `STDERRED_ESC_CODE`.
